README.md

```diff
@@ -53,4 +53,4 @@ Additionally, for ease of writing tests, this repo also contains a Go client for
 The `scripts/full_rebuild_and_run.sh` will rebuild and rerun both the initializer and controller Docker image; rerun this every time that you make a change. Arguments passed to this script will get passed to the initializer binary CLI as-is.
 
 ### Keeping Your Dev Environment Clean
-Kurtosis intentionally doesn't delete containers and volumes, which means your local Docker environment will accumulate images, containers, and volumes. Make sure to read [the Notes section of the Kurtosis README](https://github.com/kurtosis-tech/kurtosis/tree/develop#notes) for information on how to keep your local environment clean while you develop.
+Kurtosis intentionally doesn't delete containers and volumes, which means your local Docker environment will accumulate images, containers, and volumes; you can use [the script here](./scripts/clean_docker_environment.sh) to clean old containers and images. For further information, read [the Notes section of the Kurtosis README](https://github.com/kurtosis-tech/kurtosis/tree/develop#notes) for more details on how to keep your local environment clean while you develop.
```

Comment:
can you clarify which images are going to be removed? the script warning makes it clear that all containers will be removed (not just kurtosis ones) but i'm not sure which images will be removed

this script seems like it might nuke all images in the local engine (could be wrong) which might be very undesirable because some images take a long time to pull and people keep them around for convenience, or they're not linked into a remote repo (when building images locally for example) and they won't want to accidentally delete those images

Reply:
Added in the script itself